docker/php/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6-cli
```

Comment:
Travis runs on PHP 7.0, so I think that the prod runs on PHP 7 too, not on 5.6

Reply:
I'm using that since the composer.json says `"php": ">=5.6"`. Since this is OSS, it would mean a BC otherwise...